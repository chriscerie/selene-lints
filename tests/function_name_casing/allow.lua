local function Foo() end
local function foo() end

local function Bar() end
local function bar() end

local function Baz() end
local function baz() end

function x.Foo() end
function x.foo() end

function x.Bar() end
function x.bar() end

function x.Baz() end
function x.baz() end

function x:Foo() end
function x:foo() end

function x:Bar() end
function x:bar() end

function x:Baz() end
function x:baz() end

Foo()
foo()
Bar()
bar()
Baz()
baz()
