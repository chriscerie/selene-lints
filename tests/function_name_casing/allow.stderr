error[function_name_casing]: function names must start with a lowercase letter
  ┌─ allow.lua:4:16
  │
4 │ local function Bar() end
  │                ^^^
  │
  = try: bar(...) instead

error[function_name_casing]: function names must start with a lowercase letter
   ┌─ allow.lua:13:12
   │
13 │ function x.Bar() end
   │            ^^^
   │
   = try: bar(...) instead

error[function_name_casing]: function names must start with a lowercase letter
   ┌─ allow.lua:22:12
   │
22 │ function x:Bar() end
   │            ^^^
   │
   = try: bar(...) instead

